external/llvm-project/mlir/test/Integration/Dialect/Arith/CPU/test-wide-int-emulation-addi-i16.mlir

```diff
@@ -4,14 +4,14 @@
 // RUN: mlir-opt %s --convert-scf-to-cf --convert-cf-to-llvm --convert-vector-to-llvm \
 // RUN:             --convert-func-to-llvm --convert-arith-to-llvm | \
 // RUN:   mlir-cpu-runner -e entry -entry-point-result=void \
-// RUN:                   --shared-libs=%mlir_lib_dir/libmlir_c_runner_utils%shlibext | \
+// RUN:                   --shared-libs=%linalg_test_lib_dir/%prefix_mlir_c_runner_utils%shlibext | \
```

Comment:
Ok, having looked, what you actually want is to pull https://reviews.llvm.org/D143925 onto our repository instead of reinventing it badly

Reply:
(That might not be the exact commit but still)